src/main/steps/sequence/check-your-answers/content.ts

```diff
@@ -0,0 +1,68 @@
+import { startCase } from 'lodash';
+
+import { Case, YesOrNo } from '../../../app/case/case';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { Sections } from '../../../steps/sequence';
+import * as urls from '../../urls';
+
+export const generateContent: TranslationFn = ({ isDivorce, partner }) => {
+  const en = {
+    title: 'Check your answers',
+    soFar: 'so far',
+    sectionTitles: {
+      [Sections.AboutPartnership]: `About your ${isDivorce ? 'marriage' : 'civil partnership'}`,
```

Comment:
Shouldn't this come from the content?

Reply:
These are new section headers, which are currently not in any of the steps and they're used to group pages together see [NFDIV-306](https://tools.hmcts.net/jira/browse/NFDIV-306) which contains a list of these new section headers.